site/source/docs/porting/exceptions.rst

```diff
@@ -171,3 +171,57 @@ Using Exceptions and setjmp-longjmp Together
 ============================================
 
 See :ref:`using-exceptions-and-setjmp-longjmp-together`.
+
+
+Limitations
```

Comment:
Since both of these limitation are regarding `std::set_terminate` should we call this section `Limitations regarding std::terminate`?

Reply:
Done